src/nvim/window.c

```diff
@@ -782,26 +826,8 @@ void ui_ext_win_position(win_T *wp)
   FloatConfig c = wp->w_float_config;
   if (!c.external) {
     ScreenGrid *grid = &default_grid;
-    float row = c.row, col = c.col;
```

Comment:
Hmm this is not really correct for multigrid. Does it work to use `comp_row += grid->comp_row`  just below `int comp_row = (int)row - (south ? wp->w_height : 0);` ? This will also reuse the nested bufpos calculation already done for the parent float.

Reply:
Yes this is much better! I've updated the branch to reflect this. Is there something simular that can be done for computing the position of windows initially for #13403?

Reply:
this change still needs to be reverted.